Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -261,139 +262,62 @@ bool Renderer::CheckForHostConfigChanges()
 // Create On-Screen-Messages
 void Renderer::DrawDebugText()
 {
-  std::string final_yellow, final_cyan;
+  const float scale = ImGui::GetIO().DisplayFramebufferScale.x;
```

Comment:
I noticed this scale value is only for horizontal, do we need to have something for the vertical scale so that the values scale up appropriately?

Reply:
Fair point, it does seem a little confusing. We can just multiply it by m_framebuffer_scale instead. This was before I realized that DisplayFramebufferScale actually has no effect on imgui rendering.